### PR TITLE
[Request Feedback] Change Delete all to Delete selected

### LINKED ISSUE #3470: "Delete All" CTA message is unclear
**Describe the bug**
When selecting multiple entries in content manager (but not all entries) the multiple delete option says "Delete All"

**Expected behavior**
Option, provided all are not selected, should say "Delete Selected"

**System**
- Strapi version: v3.0.0-alpha.26.2

### PR DESCRIPTION
**Pull request by derrickmehaffy**
#### Description of what you did:

Update the `Delete all` to `Delete selected`

#### My PR is a:

- [ ] 💥 Breaking change
- [x] 🐛 Bug fix #3470
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
Closing in favor of a better solution: https://github.com/strapi/strapi/pull/3478

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "components.FiltersPickWrapper.hide": "Fermer",
   "components.LimitSelect.itemsPerPage": "Éléments par page",
   "components.Search.placeholder": "Rechercher une entrée...",
-  "components.TableDelete.delete": "Tout supprimer",
+  "components.TableDelete.delete": "Supprimer sélectionnée",
```
Comment (soupette) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: This should be a little more frenchy
```
Supprimer les éléments sélectionnés
```

Comment (Aurelsicoko) [reply packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: Like "Supprimer les éléments sélectionnés" for example, we have space we can have a long sentence.

Comment (derrickmehaffy) [reply packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: /s I do think that's a little long there @soupette

**Pull request closed without merging**